include/dynd/types/tuple_type.hpp

```diff
@@ -145,13 +147,13 @@ namespace ndt {
     }
 
     /** Makes a tuple type with the specified types */
-    static type make(const nd::array &field_types, bool variadic = false)
+    static type make(std::vector<type> field_types, bool variadic = false)
     {
       return type(new tuple_type(field_types, variadic), false);
```

Comment:
Should maybe be `new tuple_type(std::move(field_types), ...)`, to avoid the extra copy.